samples/python_interop/generating_n_random_bits/src/GenerateRandomNumbers.qs

```diff
@@ -0,0 +1,25 @@
+namespace GenerateRandom {
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Math;
+    open Microsoft.Quantum.Random;
+
+    operation GenerateRandomNumbers(min: Int, max: Int) : (Result[], Int) {
+
+        let nQubits = BitSizeI(max);
+        use qubits = Qubit[nQubits];
+
+        ApplyToEach(H, qubits);
+
+        let result = MeasureEachZ(qubits);
+
+        MResetEachZ(qubits);
```

Comment:
You shouldn't need to use two sets of measurements, MResetEachZ will do what you need in one call